spec/support/vcr_cassettes/lighthouse/veteran_verification/status/200_person_not_found_response.yml

```diff
@@ -0,0 +1,59 @@
+---
+http_interactions:
+- request:
+    method: get
+    uri: https://sandbox-api.va.gov/services/veteran_verification/v2/status/1012667145V762141
```

Comment:
Please safeguard the uri. ` <URL>/services/veteran_verification/v2/status/1012667145V762141`

Reply:
Yeah, I guess it's not "sensitive", so we should be good here. [VCR docs](https://benoittgt.github.io/vcr/#/configuration/filter_sensitive_data) on `filter_sensitive_data`.

If you do have cassettes with private urls, this is how we safeguard other URIs: https://github.com/department-of-veterans-affairs/vets-api/blob/master/config/initializers/integration_recorder.rb#L71-L80.